wordle-web/src/views/GameView.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <v-card color="grey-darken-4">
+    <v-card-title> Wordlinator </v-card-title>
+    <v-card-subtitle>This is a game about words</v-card-subtitle>
+    <v-card-text>
+      Here is some placeholder text for the game, lorem ipsum solte delore sit amet siv percum.
+    </v-card-text>
+    <v-card-actions>
+      <nav>
+        <v-btn variant="outlined" to="/" color="arabicaMint">Return to Home</v-btn>
+      </nav>
```

Comment:
```suggestion
        <v-btn variant="outlined" to="/" color="arabicaMint">Return to Home</v-btn>
```
You don't need to wrap this in a nav tag